main.go

```diff
@@ -110,6 +116,33 @@ sfdeploy.serverurl = https://login.salesforce.com
 sfdeploy.maxPoll = 20`
 
 func main() {
+	var projectPath string
+	flag.Parse()
+
+	if *dFlag {
+		fmt.Println("Where would you like to create new ANT projects? (ex: '/Users/username/documents/'")
+
+		customProjectPath, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		checkErr(err)
+		customProjectPath = strings.Replace(customProjectPath, "\n", "", -1)
+
+		/*
+		 * TODO: validation of the user entered path???
+		 */
+
+		os.Setenv(envPath, customProjectPath)
```

Comment:
Consider removing this.

Reply:
I'm actually working on adding the validation with strings.HasPrefix() and HasSuffix()